src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -133,18 +141,30 @@ const WorkspaceDashboard = _.flow(
   const [bucketLocation, setBucketLocation] = useState(undefined)
   const [bucketLocationType, setBucketLocationType] = useState(undefined)
 
+  const persistenceId = `workspaces/${namespace}/${name}/dashboard`
+
   const signal = useCancellation()
 
   const refresh = () => {
     loadSubmissionCount()
     loadStorageCost()
+    loadBucketSize()
     loadConsent()
     loadWsTags()
     loadBucketLocation()
   }
 
   useImperativeHandle(ref, () => ({ refresh }))
 
+  const [workspaceInfoPanelOpen, setWorkspaceInfoPanelOpen] = useState(() => getLocalPref(persistenceId)?.workspaceInfoPanelOpen)
```

Comment:
Should this have a default boolean value like the others?

Reply:
This one is a bit of a special case because it's always expanded by default. I preferred to leave it undefined and do the handling for this in the component props instead